CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## [1.52.1](https://github.com/brave/brave-ios/releases/tag/v1.52.1)
+
+- Resolved web compatibility issue on YouTube desktop site. ([#7714](https://github.com/brave/brave-ios/issues/7714))
```

Comment:
Same comment as above, looks like both issues are using the same GH issue/link. Probably copy/paste error. Regarding the note, how about this? Either way works though.

```
* Resolved web compatibility issue on YouTube (desktop version of website).
```

Makes it a bit clearer I think.

Reply:
Great, thanks